### PR TITLE
Lcm component test

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/894/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1707/ : FAILURE in 2m 19s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1968/ : FAILURE in 2m 42s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/145/ : SUCCESS in 8m 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/149/ : SUCCESS in 1m 22s

File: lib/gooddata/models/schedule.rb [hunk 1]
```diff
@@ -67,15 +67,18 @@ def create(process_id, trigger, executable, options = {})
...
+        is_dataload_process = process.type == :dataload
 
         if is_dataload_process
           dataload_datasets = options[:dataload_datasets] || options['GDC_DATALOAD_DATASETS']
           dataload_datasets = '[]' unless dataload_datasets
 
           de_synchronize_all = options[:de_synchronize_all] || options['GDC_DE_SYNCHRONIZE_ALL']
         else
-          fail 'Executable has to be provided' if executable.blank?
+          lcm_component = process.type == :lcm
```
Comment (kubamahnert) [inline lib/gooddata/models/schedule.rb]: what about processes of other component types (etl..) that also do not have an executable?

**Pull request merged**